app/scripts/controllers/eventDetails.js

```diff
@@ -690,6 +693,17 @@ angular
       }
     });
 
+    $scope.$watch(
+      'conference.locationCountry',
```

Comment:
If the country changes, we should just wipe the values for state and zip so avoid any possible confusion and allow the user to input was they need for their specific case.